Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/MaterialComponentController.cpp

```diff
@@ -560,6 +568,13 @@ namespace AZ
                     materialIt->second.m_propertyOverrides.empty() &&
                     materialIt->second.m_matModUvOverrides.empty())
                 {
+                    // TODO: Although this works, we need to make sure that the previous material AssetId
+                    // is not being used in other slots before Disconnecting from the AssetBus.
+                    //const auto& prevAssetId = materialIt->second.m_materialAsset.GetId();
+                    //if (prevAssetId.IsValid())
+                    //{
+                    //    AZ::Data::AssetBus::MultiHandler::BusDisconnect(prevAssetId);
+                    //}
```

Comment:
Todos, immediate follow up or ticket?

Reply:
I'll follow up immediately, but need to consult with @gadams3.